test/cri-containerd/createcontainer_test.go

```diff
@@ -812,14 +812,92 @@ func Test_CreateContainer_Mount_File_LCOW(t *testing.T) {
 	runCreateContainerTest(t, lcowRuntimeHandler, request)
 }
 
+func Test_CreateContainer_Mount_ReadOnlyFile_LCOW(t *testing.T) {
+	testutilities.RequiresBuild(t, osversion.V19H1)
```

Comment:
Is this check necessary?

Reply:
Not sure, I just copied this from the existing File/Dir mount tests. I guess the question is if 9P works on RS5.

Reply:
The matching dir version of this test doesn't have the restriction. Was that intentional? Can we test that out?

Reply:
That does ring a bell... I think it was that the restrict file access functionality is only in 19H1+. I think this needs to be here, but can follow up.

Reply:
Sounds good!

Reply:
Confirmed this is not present in RS5 (unless it was backported).